roles/tendrl-ansible.tendrl-server-cleanup/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+# Task file for storage nodes
+
+- name: Stop tendrl services
+  service:
+    name: tendrl-node-agent
+    state: stopped
+
+- name: Stop etcd service
+  service:
+    name: etcd
+    state: stopped
+
+- name: Stop grafana-server service
+  service:
+    name: grafana-server
+    state: stopped
+
+- name: Stop carbon-cache service
+  service:
+    name: carbon-cache
+    state: stopped
+
+- name: Stop httpd service
+  service:
+    name: httpd
+    state: stopped
+
+- name: Cleanup etcd data
+  file:
+    path: "{{etcd_db_path}}/"
+    state: absent
+
+- name: Cleanup grafana data
```

Comment:
same as above

Reply:
Here as well, I feel we are using default location to save tendrl data.

Reply:
what if someone is having other dashboards on grafana which are not related to tendrl. By cleaning grafana I think those will be lost too and then there will be no recovery for it.

Reply:
Here as well we can try to remove only tendrl specific dashboards and keep grafana-server running. Need to verify though.